Marlin/ultralcd.cpp

```diff
@@ -877,13 +879,15 @@ void kill_screen(const char* lcd_msg) {
 
     float lcd_z_offset_edit() {
       lcd_goto_screen(_lcd_z_offset_edit);
+      defer_return_to_status = true;
```

Comment:
I removed these `defer_return_to_status = true` lines from cases like this one, where it's already jumping straight to a screen handler that already sets `defer_return_to_status = true`.